easybuild/easyconfigs/q/qmflows/qmflows-1.0.0_version.patch

```diff
@@ -0,0 +1,25 @@
+During EasyBuild, tool.setuptools_scm appears to rewrite '__version__' in _version.py
+into 'version', leading to import issues.
```

Comment:
please add author info to the patch?
and don't forget to update the checksum :)